### PR TITLE
Update spec for multiply operation

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - Update spec for multiply operation

### Tickets:
 - 46006

### CODE REVIEW

File: docs/ops/arithmetic/Multiply_1.md [hunk 1]
```diff
@@ -4,48 +4,52 @@
...
 **Category**: Arithmetic binary operation
 
-**Short description**: *Multiply* performs element-wise multiplication operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *Multiply* performs element-wise multiplication operation with two given tensors applying broadcasting rule specified in the *auto_broacast* attribute.
+
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Multiply* performs multiplication operation for the input tensors *a* and *b* using the formula below:
+
```
Comment (lazarevevgeny) [inline docs/ops/arithmetic/Multiply_1.md:13]: redundant empty line

File: docs/ops/arithmetic/Multiply_1.md [hunk 2]
```diff
@@ -4,48 +4,52 @@
...
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *none* - no auto-broadcasting is allowed, all input shapes must match,
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
+    * *pdpd* - PaddlePaddle-style implicit broadcasting, description available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
```
Comment (lazarevevgeny) [inline docs/ops/arithmetic/Multiply_1.md:27]: `description is available`

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/ops/arithmetic/Multiply_1.md (hunk 1 above)
Comment (pelszkow) [reply docs/ops/arithmetic/Multiply_1.md:13]: Should fix it for Add op too?

File: docs/ops/arithmetic/Multiply_1.md (hunk 2 above)
Comment (pelszkow) [reply docs/ops/arithmetic/Multiply_1.md:27]: Should fix it for Add op too?

**Review by sdurawa — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**